composer.json

```diff
@@ -34,7 +34,8 @@
         "symfony/yaml" : "2.8.42",
         "twig/twig" : "1.35.3",
         "vlucas/phpdotenv" : "2.4.0",
-        "zendframework/zendframework" : "2.5.3"
+        "zendframework/zendframework" : "2.5.3",
+        "yubico/u2flib-server": "^1.0"
```

Comment:
rec hard-coding full version in (ie. change the `^1.0` to `1.0.2`"

Reply:
OK. This was the result of "composer require yubico/u2flib-server". Guess I'm not fully versed on best practices here.

Reply:
I'd be glad to bring this in (since we also are alphabetizing the listings in composer.json) rather than have you spend time on this. Just let me know.

Reply:
OK feel free to do a follow-up commit.